content/ja/community/_index.html

```diff
@@ -0,0 +1,236 @@
+---
+title: コミュニティ
+layout: basic
+cid: community
+---
+
+<div class="newcommunitywrapper">
+<div class="banner1">
+  <img src="/images/community/kubernetes-community-final-02.jpg" alt="Kubernetes Conference Gallery" style="width:100%;padding-left:0px" class="desktop">
+  <img src="/images/community/kubernetes-community-02-mobile.jpg" alt="Kubernetes Conference Gallery" style="width:100%;padding-left:0px" class="mobile">
```

Comment:
Optional: localize the `alt` attribute.

Reply:
Right, I'd better translate the `alt` attribute for better accessibility.